src/main/resources/com/welovecoding/nbeditorconfig/EditorConfigExample.editorconfig

```diff
@@ -0,0 +1,33 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
This file is used for preview in Fonts and Colors option. (i.e. this is not a file for template) So I think that  we should not remove it.